Sources/Plasma/PubUtilLib/plPhysX/plSimulationMgr.h

```diff
@@ -163,28 +161,34 @@ class plSimulationMgr : public hsKeyedObject
 #ifndef PLASMA_EXTERNAL_RELEASE
     void IDrawActiveActorList();
 #endif //PLASMA_EXTERNAL_RELEASE
+
+public:
+    // Output the given debug text to the simulation log.
+    template<typename... _Args>
+    static void Log(const char* formatStr, _Args&&... args)
+    {
+        if (GetInstance()) {
+            if (GetInstance()->fLog)
+                GetInstance()->fLog->AddLine(formatStr, std::forward<_Args>(args)...);
+        }
+    }
 };
 
 #define SIM_VERBOSE
 
 #ifdef SIM_VERBOSE
-#include <cstdarg>     // only include when we need to call plSimulationMgr::Log
-
-inline void SimLog(const char *str, ...)
+template<typename... _Args>
+inline void SimLog(const char* format, _Args&&... args)
 {
-    va_list args;
-    va_start(args, str);
-    plSimulationMgr::LogV(str, args);
-    va_end(args);
+    plSimulationMgr::Log(format, std::forward<_Args>(args)...);
 }
 
 #else
-
+template<typename... _Args>
 inline void SimLog(const char *str, ...)
```

Comment:
In order for this to get called as a template, you'll need to change the C varargs to C++ varargs...